### ISSUE TITLE
[editorial] grammar issues in vocabulary/index.html

### ISSUE THREAD

**Issue by TallTed** [w3c/activitystreams#624]
https://github.com/w3c/activitystreams/blob/6880c92381cd5e3518651a1135ed819ef4828368/vocabulary/index.html#L4383-L4387

Language needs a bit of correction.

```
            One or more "tags" that have been associated with an objects.
            A tag can be any kind of Object. The key difference between
            <code>attachment</code> and <code>tag</code> is that the
            former implies association by inclusion, while the latter
            implies associated by reference.
```

* `an objects` in the first line is being addressed through #621 (changing to `an object`).

* `associated by` in the last line should be `association by`, matching the preceding line.

* I would suggest also changing the verb tense in the first line, from `have been associated` to `are associated`

**Comment by evanp** [comment]
Agreed on all this. I think we have another ticket open for "an objects", but this is more thorough. Also, a normative difference -- a tag can be any kind of Object **or Link**.

**Comment by TallTed** [comment]
> I think we have another ticket open for "an objects"

Yes, that's the cited #621.

**Comment by evanp** [comment]
We have an erratum PR open for this issue: https://github.com/w3c/activitystreams/pull/626

**Comment by TallTed** [comment]
#626 doesn't handle my third bullet.

**Comment by evanp** [comment]
Ted, can you live with correcting the clear errors, and making the editorial change, which seems harmless, in our next version of the doc?

**Comment by evanp** [comment]
Errata-ed and editors-drafted. Thanks @TallTed !
